### PR TITLE
Remove explicit repo name loading local bzl files

### PR DESCRIPTION
**Pull request by erikkerber**
Removes explicit repo names in load commands in favor of relative names.

### CODE REVIEW

File: apple/cc_toolchain_forwarder.bzl [hunk 1]
```diff
@@ -16,8 +16,8 @@
 A rule for handling the cc_toolchains and their constraints for a potential "fat" Mach-O binary.
 """
 
-load("@build_bazel_rules_apple//apple/internal:providers.bzl", "new_appleplatforminfo")
-load("@build_bazel_rules_apple//apple:providers.bzl", "ApplePlatformInfo")
+load("//apple/internal:providers.bzl", "new_appleplatforminfo")
+load("//apple:providers.bzl", "ApplePlatformInfo")
 load("@bazel_tools//tools/cpp:toolchain_utils.bzl", "find_cpp_toolchain", "use_cpp_toolchain")
```
Comment (brentleyjones) [inline apple/cc_toolchain_forwarder.bzl:21]: After doing this, can you reorder the loads? Relative ones should be below external ones.

Comment (erikkerber) [reply apple/cc_toolchain_forwarder.bzl:21]: Related: I noticed buildifier isn't run on PR's here. By design or should that be added?

Comment (brentleyjones) [reply apple/cc_toolchain_forwarder.bzl:21]: It's checked in `buildkite/rules-apple-darwin/pr/buildifier`, right?

Comment (brentleyjones) [reply apple/cc_toolchain_forwarder.bzl:21]: (Though maybe it's not as strict as it should be.)

Comment (erikkerber) [reply apple/cc_toolchain_forwarder.bzl:21]: Ran buildifier on the repo locally/manually and got this diff and others like it. I'll look to see if it's a version diff or something.

CleanShot 2024-07-11 at 14 43 48@2x

Comment (erikkerber) [reply apple/cc_toolchain_forwarder.bzl:21]: I'll make the sorts another PR, and if I see a buildifier fix I'll include that with it.

**Comment by erikkerber** [comment]
`last_green`, and the UI test sim launch failure I believe are unrelated.

**Review by brentleyjones — APPROVED** [review]

**Pull request merged**